lib/models/config.ts

```diff
@@ -254,6 +323,26 @@ export default (crowi: Crowi) => {
     return localConfig
   }
 
+  configSchema.statics.migrate = async function() {
```

Comment:
Shouldn't the migration function check the current status which version is running previously like if `aws:bucket` exists means the version < v1.7 ?
It is a bit danger if always migration function run without any check IMO.

Reply:
`Config.migrate` method is idempotent because of `Config.copyByParams` method checks if `oldKey` exists. So I think that checking `aws:bucket` is redundant.

By the way, in essence, it should manage the migration with a database.